src/plugin/plugin.ts

```diff
@@ -898,13 +913,25 @@ class Purchases {
   /**
    * Enables/Disables debugs logs
    * @param {boolean} enabled Enable or not debug logs
+   * @deprecated Use {@link setLogLevel} instead.
    */
   public static setDebugLogsEnabled(enabled: boolean): void {
     window.cordova.exec(null, null, PLUGIN_NAME, "setDebugLogsEnabled", [
       enabled,
     ]);
   }
 
+  /**
+   * Used to set the log level. Useful for debugging issues with the lovely team @RevenueCat.
+   * @param {boolean} enabled Enable or not debug logs
```

Comment:
This should be updated

Reply:
also slight copy suggestion: 
```suggestion
   * @param {boolean} enabled Enable or disable debug logs
```

Reply:
Oops thanks.

Reply:
Fixed.